app/templates/public.hbs

```diff
@@ -44,7 +44,7 @@
         {{outlet}}
       </div>
       {{/unless}}
-      <div class="three wide column">
+      <div class="three wide column" style="overflow-x: hidden;">
```

Comment:
Why not add a class?

Reply:
Please suggest if there is an existing class or I need to add a new one.

Reply:
I think adding a new class is the way. I am working on this.

Reply:
I have added a new commit to solve this [https://github.com/fossasia/open-event-frontend/pull/6582/commits/953b8a68082ea7fed713fbdfe55eeb642e2f9e76](url)

Reply:
I have added a new commit to solve this. [https://github.com/fossasia/open-event-frontend/pull/6582/commits/953b8a68082ea7fed713fbdfe55eeb642e2f9e76](url)

Reply:
I have added a new commit to solve this issue.